### PR TITLE
Fp 3785

### PR DESCRIPTION
**Pull request by davidallardsb**
#### What does this PR do? (please provide any background)
Reviews branding on SBP and styles accordingly for Warwick Castle.

#### What tests does this PR have?
None 

#### How can this be tested?
Make sure Warwick Castle on SBP is styled as per the Warwick brand. 

#### Any tech debt?
Nope 

#### What gif best describes how you feel about this work?

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

File: brands/alton-towers/src/less/forms.less [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import '../../../brand-globals/src/less/forms';
 
-.payment-form-wrapper .fa-lock:before {
-  color: @brand-primary;
+.payment-form-wrapper h1 svg {
+  fill: @brand-primary;
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/forms.less]: try `fill:currentColor`

File: brands/warwick/src/less/forms.less [hunk 2]
```diff
@@ -1 +1,39 @@
 @import '../../../brand-globals/src/less/forms';
+
+.payment-form-wrapper h1 svg {
+  fill: @brand-yellow;
```
Comment (jodiedoubleday) [inline brands/warwick/src/less/forms.less]: if you make all svgs `fill:currentColor` it should solve this for all svg's

File: brands/warwick/src/less/forms.less [hunk 3]
```diff
@@ -1 +1,39 @@
...
+}
+
+legend {
+  margin: @padding-large-horizontal 0;
+  border-top: 1px solid @gray-lighter;
+  padding-top: @padding-large-horizontal/2;
+}
+
+form {
+  .fa-lock:before {
```
Comment (jodiedoubleday) [inline brands/warwick/src/less/forms.less]: We shouldn't I have fa-lock in SBP anymore as we removed font-awesome didn't we?

File: brands/warwick/src/less/forms.less [hunk 4]
```diff
@@ -1 +1,39 @@
...
+    border: 2px solid @brand-danger;
+  }
+  .help-block {
+    z-index: 0;
+    color: @white;
+    font-size: 12px;
+    position: relative;
+    margin: -4px 0px 0 0;
+    display: inline-block;
+    padding: 6px 6px 2px 6px;
```
Comment (jodiedoubleday) [inline brands/warwick/src/less/forms.less]: could use padding variables here.

File: brands/warwick/src/less/labels.less [hunk 5]
```diff
@@ -1 +1,5 @@
 @import '../../../brand-globals/src/less/labels';
+
+label {
+  color: @label-color;
```
Comment (jodiedoubleday) [inline brands/warwick/src/less/labels.less]: isn't this a default think, seems weird to have label color as the label color.

File: brands/alton-towers/src/less/forms.less (hunk 1 above)
Comment (davidallardsb) [reply brands/alton-towers/src/less/forms.less]: That won't work in this case as I am using a different colour from the parent :)

image

File: brands/warwick/src/less/forms.less (hunk 2 above)
Comment (davidallardsb) [reply brands/warwick/src/less/forms.less]: As above :)

File: brands/warwick/src/less/navs.less [hunk 6]
```diff
@@ -9,9 +9,19 @@
...
-  &:hover,
-  &:focus {
-    border-color: @nav-tabs-link-hover-border-color @nav-tabs-link-hover-border-color transparent;
+.nav-tabs {
+  border-bottom: none;
+  > li > a {
+    color: @white;
+    border-color: @body-bg;
+    background: @btn-default-bg;
+    &:HOVER {
```
Comment (jodiedoubleday) [inline brands/warwick/src/less/navs.less]: CAPITALS

**Pull request merged**